Recrutify/Recrutify.Host/Configuration/Profiles/CandidateProfile.cs

```diff
@@ -18,7 +18,11 @@ public CandidateProfile()
                 .ForMember(dest => dest.IsAssigned, conf => conf.MapFrom(src => false));
             CreateMap<ProjectResult, ScheduleCandidateProjectResultDTO>().ReverseMap()
                 .ForMember(dest => dest.PrimarySkill, conf => conf.MapFrom(src => new CandidatePrimarySkillDTO { Id = src.PrimarySkill.Id, Name = src.PrimarySkill.Name }));
-            CreateMap<CandidatePrimarySkill, CandidatePrimarySkillDTO>().ReverseMap();
+            CreateMap<CandidatePrimarySkill, CandidatePrimarySkillDTO>()
+                .ForMember(dest => dest.Name, conf => conf.MapFrom(src => new CandidatePrimarySkill { Id = src.Id, Name = src.Name }));
```

Comment:
ignore

Reply:
remove .ForMember(dest => dest.Name, conf => conf.MapFrom(src => new CandidatePrimarySkill { Id = src.Id, Name = src.Name })); it at all